core/storage/merkle/polkadot_trie_db/polkadot_codec.cpp

```diff
@@ -83,24 +84,23 @@ namespace kagome::storage::merkle {
     return out;
   }
 
-  common::Buffer PolkadotCodec::keyToNibbles(const common::Buffer &key) const {
+  common::Buffer PolkadotCodec::keyToNibbles(const common::Buffer &key) {
     auto nibbles = key.size() * 2;
 
     // if last nibble in `key` is 0
-    bool last_nibble_0 =
-        !key.empty() && (high4nibbles(key[key.size() - 1]) == 0);
+    bool last_nibble_0 = !key.empty() && (highNibble(key[key.size() - 1]) == 0);
```

Comment:
apply clang-format

Reply:
I did. Mine doesn't change this line. It fits exactly in 80 symbols.